kuma/wiki/utils.py

```diff
@@ -64,7 +64,11 @@ def get_doc_components_from_url(url, required_locale=None, check_host=True):
     # Extract locale and path from URL:
     parsed = urlparse(url)  # Never has errors AFAICT
     if check_host and parsed.netloc:
-        return False
+        # If its not redirecting to our domain,
```

Comment:
Nit: Change ``its`` to ``it's``. The contraction ``it's`` is used when it can be replaced by ``it is`` or ``it has``. The possessive ``its`` is used for ownership like "I tracked the horse by following its hoof prints", in a similar way to "his hoof prints" or "her hoof prints".